lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js

```diff
@@ -769,10 +755,12 @@
           g.selectAll("text")
             .attr("fill", "#d8d8d8");
 
-        // Initialize the brush component with pretty resize handles.
-        var gBrush = g.append("g").attr("class", "brush").call(brush);
-        gBrush.selectAll("rect").attr("height", height);
-        gBrush.selectAll(".resize").append("path").attr("d", resizePath);
+          // Initialize the brush component with pretty resize handles.
```

Comment:
Same here: is the indentation inconsistent?

Reply:
Yes. Same here. Just a small adjustment to make it line up better.